src/NetAnalyzers/UnitTests/Microsoft.NetCore.Analyzers/Performance/UseStringContainsCharOverloadWithSingleCharactersTests.cs

```diff
@@ -0,0 +1,210 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.Testing;
+using Xunit;
+using VerifyCS = Test.Utilities.CSharpCodeFixVerifier<
+    Microsoft.NetCore.Analyzers.Performance.UseStringContainsCharOverloadWithSingleCharactersAnalyzer,
+    Microsoft.NetCore.CSharp.Analyzers.Performance.CSharpUseStringContainsCharOverloadWithSingleCharactersFixer>;
+using VerifyVB = Test.Utilities.VisualBasicCodeFixVerifier<
+    Microsoft.NetCore.Analyzers.Performance.UseStringContainsCharOverloadWithSingleCharactersAnalyzer,
+    Microsoft.NetCore.VisualBasic.Analyzers.Performance.BasicUseStringContainsCharOverloadWithSingleCharactersFixer>;
+
+namespace Microsoft.NetCore.Analyzers.Performance.UnitTests
+{
+    public class UseStringContainsCharOverloadWithSingleCharactersTests
+    {
+        [Fact]
+        public async Task CSharp_RegularStringLiteral_Fixed()
+        {
+            var violatingSourceCode = @"using System;
+public class TestClass
+{
+    public bool TestMethod(string input)
+    {
+        return ""testString"".Contains([|""a""|]);
```

Comment:
Probably not very important, but consider adding a test for constant interpolated strings, i.e `return "testString".Contains($"a");` or `"testString.Contains($"{CONST_CHAR_DEFINED_ABOVE}");`

Reply:
That use case was specifically excluded in the source issue.

Reply:
Unless roslyn somehow optimizes constant values in a way that the interpolation with a constant value would end up classifying as a `LiteralSyntax`?

Reply:
@MeikTranel I couldn't see anything about it in the comments. Was that said in the API review video?